### DISCUSSION TITLE
打包后为何vite代理未生效，但在dev环境下生效

### DISCUSSION THREAD

**Discussion by sxgan-code** [alex8088/electron-vite#582] [Q&A]
按照vite的官网进行了代理配置，开发环境一切正常，但是最近开发完成后进行打包，所有接口都无法被代理了

**Comment by alex8088** [comment]
vite的开发server，只为开发服务，无论是electron开发还是普通web开发，都不会作用生产，vite本质是构建工具。

**Comment by sxgan-code** [comment]
我是后端开发，对前端还不太了解，感谢解答，我去码代码了
________________________________
发件人: ***@***.*** ***@***.***> 代表 alex.wei ***@***.***>
发送时间: 2024年8月6日 07:00
收件人: alex8088/electron-vite ***@***.***>
抄送: Zoey ***@***.***>; Author ***@***.***>
主题: Re: [alex8088/electron-vite] 打包后为何vite代理未生效，但在dev环境下生效 (Discussion #582)


vite的开发server，只为开发服务，无论是electron开发还是普通web开发，都不会作用生产，vite本质是构建工具。

―
